gowin/arch.cc

```diff
@@ -0,0 +1,1248 @@
+/*
+ *  nextpnr -- Next Generation Place and Route
+ *
+ *  Copyright (C) 2018  Clifford Wolf <clifford@symbioticeda.com>
```

Comment:
Please update and add your own name

Reply:
Should I do anything with @clairexen's entry? Seems to be using her deadname across the whole project except in _one_ location, so not sure what the accepted approach is for a new copy of old code.

Reply:
Yes, update it in new code (afaik Yosys is mostly doing the same)